scipy/signal/_signaltools.py

```diff
@@ -1555,7 +1567,23 @@ def medfilt(volume, kernel_size=None):
 
     numels = np.prod(kernel_size, axis=0)
     order = numels // 2
-    return _sigtools._order_filterND(volume, domain, order)
+
+    if volume.dtype in [np.bool_, np.cfloat, np.cdouble,
+                                       np.clongdouble, np.float16,]:
```

Comment:
something is off with indentation here, and on line 1584 below

Reply:
Fixed in the last commit.